packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationProfileRoutes.tsx

```diff
@@ -12,65 +14,87 @@ import { VerifiedDomainPage } from './VerifiedDomainPage';
 import { VerifyDomainPage } from './VerifyDomainPage';
 
 export const OrganizationProfileRoutes = (props: PropsOfComponent<typeof ProfileCardContent>) => {
+  const { pages } = useOrganizationProfileContext();
+  const isMembersPageRoot = pages.routes[0].id === 'members';
+  const isSettingsPageRoot = pages.routes[0].id === 'settings';
+  const isPredefinedPageRoot = isSettingsPageRoot || isMembersPageRoot;
+
   return (
     <ProfileCardContent contentRef={props.contentRef}>
-      <Route path='organization-settings'>
-        <Switch>
+      <Switch>
+        {/* Custom Pages */}
+        {pages.contents?.map((customPage, index) => (
           <Route
-            path='profile'
-            flowStart
+            index={!isPredefinedPageRoot && index === 0}
+            path={!isPredefinedPageRoot && index === 0 ? undefined : customPage.url}
```

Comment:
Can we extract this condition and give it some context with a good variable name?